opal/mca/pmix/pmix112/configure.m4

```diff
@@ -43,7 +43,7 @@ AC_DEFUN([MCA_opal_pmix_pmix112_CONFIG],[
                   [AC_HELP_STRING([--enable-pmix-dstore],
                                   [Enable PMIx shared memory data store (default: disabled)])])
     AC_MSG_CHECKING([if PMIx shared memory data store is enabled])
-    if test "$enable_pmix3_dstore" == "yes"; then
+    if test "$enable_pmix_dstore" = "yes"; then
```

Comment:
Let's change this selection statement to match `master` to make it easier to compare the branches:
```m4
    if test "$enable_pmix_dstore" != "no"; then
```